composer.json

```diff
@@ -18,22 +18,12 @@
     "issues": "https://github.com/stayallive/wp-sentry/issues"
   },
   "autoload": {
-    "psr-0": {
-      "WP_Sentry_": "."
-    }
+    "classmap": [
+      "src/"
+    ]
   },
   "require": {
     "php": ">=5.4.0",
-    "composer/installers": "~1.0",
```

Comment:
Please keep `composer/installers` as a dependency, it will be a useless composer plugin without it.

Reply:
If you need some examples:
* https://github.com/algolia/algoliasearch-wordpress/blob/0d3c256d3a93acdc134c90a78ccb6e01246e2824/composer.json
* https://github.com/Automattic/jetpack/blob/9c1c52ed99f316d489ae1db539d8d778a325a605/composer.json
* https://github.com/roots/soil/blob/d0e409e70dafff4ceeaa8ef941ee319e350317dd/composer.json

Reply:
Sorry about that. Will put this one back. Somehow thought it was for phpcs.

Reply:
OK, thanks! 🎉 
BTW don't forget to publish it on [packagist](https://packagist.org/?) after. 😉
Thanks again!

Reply:
Yes 👍  Done as 2.1.1/2.1.2. Should be OK now. Thanks for PR'ing this 💯